### PR TITLE
Nv 2612 - Tenant Page Empty State

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### What change does this PR introduce?

Display an empty state when no tenants are found

image

### CODE REVIEW

File: apps/web/src/design-system/icons/general/Buildings.tsx [hunk 1]
```diff
@@ -2,12 +2,19 @@ import React from 'react';
...
   return (
-    <svg width="20" height="18" viewBox="0 0 20 18" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <svg
+      width="20"
+      height="18"
+      viewBox="0 0 20 18"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+      style={{ ...(props.style || {}), fill: props.style?.fill || 'currentColor' }}
```
Comment (ainouzgali) [inline apps/web/src/design-system/icons/general/Buildings.tsx:12]: Would just adding color inside the Building component not work?

**Review by ainouzgali — APPROVED** [review]
Added a few suggestions❤️

File: apps/web/src/pages/tenants/components/list/TenantsList.tsx [hunk 2]
```diff
@@ -23,9 +26,12 @@ export const TenantsList = ({ onAddTenantClick }: { onAddTenantClick: React.Mous
       <Container fluid sx={{ padding: '0 30px 8px 30px' }}>
         <Toolbar onAddTenantClick={onAddTenantClick} tenantLoading={tenantLoading} />
       </Container>
-      <When truthy={hasTenants || isLoading}>
+      <When truthy={loadingPhase}>
```
[the remark this answers is not in the document]
Comment (djabarovgeorge) [reply apps/web/src/pages/tenants/components/list/TenantsList.tsx:29]: for me, one boolean is more readable as long as it is chosen right. 🙃

File: apps/web/src/design-system/icons/general/Buildings.tsx (hunk 1 above)
Comment (djabarovgeorge) [reply apps/web/src/design-system/icons/general/Buildings.tsx:12]: it did not work for me but let me double-check it! :)

File: apps/web/src/pages/tenants/components/list/TenantsListNoData.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  color: ${colors.B40};
+  margin: 0;
+`;
+
+export const TenantsListNoData = () => {
+  return (
+    <NoDataHolder data-test-id="no-integrations-placeholder">
+      <Buildings style={{ fill: colors.B30, width: '50px', height: '45px', margin: '30px' }} />
+      <NoDataText>Add the first tenant for the</NoDataText>
+      <NoDataText>development environment</NoDataText>
```
[the remark this answers is not in the document]
Comment (djabarovgeorge) [reply apps/web/src/pages/tenants/components/list/TenantsListNoData.tsx:27]: Oh really? I thought about it as a general 'development' tag, thanks for bringing it up I will update it!

**Pull request merged**